<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label7">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>530</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Button9">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>530</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="OSideButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>390</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>O</string>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>310</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:3px solid color;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label6">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>420</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:3px solid black;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>310</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:3px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Button1">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>310</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>500</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Your Side:(X goes first)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>310</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:3px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Button7">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>530</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:3px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button4">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>420</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button8">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>530</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button2">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>310</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button5">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>420</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label5">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>420</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:3px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label4">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>420</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:3px solid black;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label8">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>530</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:3px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="XSideButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>390</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button3">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>310</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label9">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>530</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border:3px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Button6">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>420</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>510</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <pointsize>15</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Reset Board</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>580</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <pointsize>15</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Exit Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="goBack1v1Button">
    <property name="geometry">
     <rect>
      <x>1330</x>
      <y>440</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Go Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>320</y>
      <width>421</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mistral</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>All the Very Best !!</string>
    </property>
   </widget>
   <zorder>label1</zorder>
   <zorder>Button1</zorder>
   <zorder>label9</zorder>
   <zorder>label5</zorder>
   <zorder>label4</zorder>
   <zorder>label8</zorder>
   <zorder>label7</zorder>
   <zorder>Button9</zorder>
   <zorder>OSideButton</zorder>
   <zorder>label6</zorder>
   <zorder>label2</zorder>
   <zorder>label</zorder>
   <zorder>label3</zorder>
   <zorder>Button7</zorder>
   <zorder>Button4</zorder>
   <zorder>Button8</zorder>
   <zorder>Button2</zorder>
   <zorder>Button5</zorder>
   <zorder>XSideButton</zorder>
   <zorder>Button3</zorder>
   <zorder>Button6</zorder>
   <zorder>reset</zorder>
   <zorder>exit</zorder>
   <zorder>goBack1v1Button</zorder>
   <zorder>resultLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
